===================================================================
RCS file: /usr/local/src/cvsroot/nefu/TDK/tdk.c,v
retrieving revision 1.4
retrieving revision 1.5
diff -u -r1.4 -r1.5
--- TDK/tdk.c	2001/05/10 21:23:32	1.4
+++ TDK/tdk.c	2003/11/04 19:40:56	1.5
@@ -1,4 +1,5 @@
 #include <sys/types.h>
+#include <sys/time.h>
 #include <netinet/in.h>
 
 #include <stdio.h>
